docs/01_nodeos/02_usage/60_how-to-guides/10_how-to-configure-state-storage.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+This how-to describes configuration of the Nodeos `backing store`. Nodeos can now use chainbase or rocks-db as a backing store for smart contract state.
```

Comment:
alternative: describes the configuration of Nodeos
chainbase, rocks-db -> Chainbase, RocksDB